src/clj/rems/poller/email.clj

```diff
@@ -31,9 +32,18 @@
   (str (:public-url env) "accept-invitation?token=" token))
 
 (defn- application-id-for-email [application]
```

Comment:
Hmm, would you be willing to change the function name to reflect the addition of application title/description? (e.g., application-id-and-description-for-email or if you come up with something better...)

Reply:
done